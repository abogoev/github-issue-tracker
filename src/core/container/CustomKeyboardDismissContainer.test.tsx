import React from "react";
import * as RN from "react-native";
import CustomKeyboardDismissContainer from "./CustomKeyboardDismissContainer";
import { create } from "react-test-renderer";

describe("CustomBox", () => {
  it("should render correctly", () => {
    const randomText = "Hello";
    const component = create(
      <CustomKeyboardDismissContainer>
        <RN.Text>{randomText}</RN.Text>
      </CustomKeyboardDismissContainer>
    );

    expect(component.root.findByType(RN.Text).props.children).toBe(randomText);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
