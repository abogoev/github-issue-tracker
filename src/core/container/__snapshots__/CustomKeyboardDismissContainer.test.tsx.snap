// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomBox should render correctly 1`] = `
<View
  accessibilityHint="Press anywhere to dismiss keyboard"
  accessibilityLabel="Keyboard Dismiss Container"
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
